### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by fonnesbeck**
A couple of updates, including adding Edward to list of packages that use PyMC3.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -124,9 +124,10 @@ Software using PyMC3
 --------------------
 
  - `Bambi <https://github.com/bambinos/bambi>`__: BAyesian Model-Building Interface (BAMBI) in Python.
- - `NiPyMC <https://github.com/PsychoinformaticsLab/nipymc>`__: Bayesian mixed-effects modeling of fMRI data in Python
- - `gelato <https://github.com/ferrine/gelato>`__: Bayesian Neural Networks with PyMC3 and Lasagne
- - `beat <https://github.com/hvasbath/beat>`__: Bayesian Earthquake Analysis Tool
+ - `NiPyMC <https://github.com/PsychoinformaticsLab/nipymc>`__: Bayesian mixed-effects modeling of fMRI data in Python.
+ - `gelato <https://github.com/ferrine/gelato>`__: Bayesian Neural Networks with PyMC3 and Lasagne.
+ - `beat <https://github.com/hvasbath/beat>`__: Bayesian Earthquake Analysis Tool.
+ - `Edward <https://github.com/blei-lab/edward>`__: A library for probabilistic modeling, inference, and criticism.
```
Comment (twiecki) [inline README.rst:130]: I don't think `Edward` supports `PyMC3` anymore, they deleted all external wrappers.

Comment (fonnesbeck) [reply README.rst:130]: There is a `PyMC3Model` class in the code base, and PyMC3 is listed as an extras requirement. I think that counts.

Comment (twiecki) [reply README.rst:130]: https://github.com/blei-lab/edward/pull/344

Comment (twiecki) [reply README.rst:130]: I suppose they're still there, just deprecated. I'm OK with this then if you want to add it.

Comment (fonnesbeck) [reply README.rst:130]: Interesting. Are we supposed to take over that code, then?

Comment (twiecki) [reply README.rst:130]: I think we should.

Comment (twiecki) [reply README.rst:130]: Want to leave this line in? Feel free to merge if so.

**Pull request merged**